### ISSUE TITLE
Design for PolygonFilter

### ISSUE THREAD

**Issue by flavour** [sahana/WACOP#18]
Implement the design for the Polygon Filter:
https://www.figma.com/file/ayeUHp99uQ3zb7XYOVjr3O9w/wacop-organisms-170619---Page-1

**Comment by flavour** [comment]
@devinbalkind 
I am working through this currently and have a question:
When the 'Draw a Map Area' button is pressed then we show the new panel (with instructions and buttons) and activate the draw control...OK so far, however currently the design shows the 'Draw a Map Area' button still visible so users might click on it again.
The simplest option for me is to simple disable the click handler on the button whilst the draw control is active, btu this still seems like poor UX which could be improved.
Options would be hiding the button or showing it as disabled:
https://uxplanet.org/button-ux-design-best-practices-types-and-states-647cf4ae0fc6#f57d

I guess the foundation "button disabled" class is what I should apply & then this style can be tweaked in CSS by Drew as-desired? (which could even meaning hide)
http://foundation.zurb.com/sites/docs/v/5.5.3/components/buttons.html#button-colors

**Comment by devinbalkind** [comment]
Yes I think "button disabled" is good and we can turn the button gray to indicate it isn't clickable.

Pinging @dhornbein.

**Comment by dhornbein** [comment]
Has there been any markup added to the theme to display the polygon button in the filter? I'm not seeing anything to style.

**Comment by flavour** [comment]
This is still pending (stashed)

**Comment by flavour** [comment]
Basically there is a lot of Events need doing...this is way more than just style...and needs to also work with the current option

**Comment by devinbalkind** [comment]
Hi. Fran, are you waiting on anything from us to move forward on this?

**Comment by flavour** [comment]
This has now been delivered to the Demo server & style can be tweaked if-necessary (I had a little camping holiday in Cornwall, hence the delay)
